src/database/sql_database.cc

```diff
@@ -2555,10 +2555,10 @@ std::unique_ptr<std::ostringstream> SQLDatabase::sqlForInsert(const std::shared_
     std::ostringstream qb;
     qb << "INSERT INTO " << TQ(tableName) << " (" << fields.str() << ") VALUES (" << values.str() << ')';
 
-    return std::make_unique<std::ostringstream>(std::move(qb));
+    return qb;
 }
```

Comment:
Isn't the stream copied here? What happens to its contents?

Maybe resolve by calling `.str()` and making the whole thing return `std::string` (I don't like the streams!)

Reply:
No this is an implicit move as the types match.